java/ql/src/utils/stub-generator/Stubs.qll

```diff
@@ -96,15 +104,14 @@ abstract private class GeneratedType extends ClassOrInterface {
   }
 
   final Type getAGeneratedType() {
-    result = this.getAnInterestingBaseType()
-    or
-    result = this.getAGeneratedMember().(Callable).getReturnType()
-    or
-    result = this.getAGeneratedMember().(Callable).getAParameter().getType()
-    or
-    result = this.getAGeneratedMember().(Field).getType()
-    or
-    result = this.getAGeneratedMember().(NestedType)
+    result = this.getAnInterestingBaseType() or
+    result = this.getAGeneratedMember().(Callable).getReturnType() or
+    result = this.getAGeneratedMember().(Callable).getAParameter().getType() or
+    result = this.getAGeneratedMember().(Field).getType() or
+    result = this.getAGeneratedMember().(NestedType) or
+    result = this.(AnnotationType).getAnAnnotation().getType() or
+    result = this.(AnnotationType).getAnAnnotation().getAValue().getType() or
+    result = this.(AnnotationType).getAnAnnotation().getAValue().(ArrayInit).getAnInit().getType()
```

Comment:
Note that `getAValue` was recently deprecated because it's usually not useful -- use `getAnArrayValue(_)` instead, which unpacks array members